llama_stack/providers/utils/inference/prompt_adapter.py

```diff
@@ -271,11 +271,15 @@ def chat_completion_request_to_messages(
         log.error(f"Unsupported inference model? {model.descriptor()}")
         return request.messages
 
-    if model.model_family == ModelFamily.llama3_1 or (
-        model.model_family == ModelFamily.llama3_2
-        and is_multimodal(model.core_model_id)
+    if (
+        model.model_family == ModelFamily.llama3_1
+        or (
+            model.model_family == ModelFamily.llama3_2
+            and is_multimodal(model.core_model_id)
+        )
+        or model.model_family == ModelFamily.llama3_3
     ):
-        # llama3.1 and llama3.2 multimodal models follow the same tool prompt format
+        # llama3.1, llama3.2 multimodal and llama3.3 models follow the same tool prompt format
```

Comment:
Needed otherwise tool calls don't work. Prompt template for 3.3 is the same as 3.1: https://www.llama.com/docs/model-cards-and-prompt-formats/llama3_3/

Reply:
nit: `model.model_family in (llama3_1, llama3_3)` ?

Reply:
this is a good call out. the prompt format for 3.3 is actually much closer to 3.2 -- I will update the model card documentation today.

Reply:
I've just tested it using the prompt format for python_list. nice! It works a lot more consistently. Getting a lot higher quality and consistent tool calls.

Reply:
Updated.